src/trails.erl

```diff
@@ -96,38 +110,51 @@ trail(PathMatch, ModuleHandler, Options, MetaData, Constraints) ->
    , constraints => Constraints
    }.
 
+%% @doc Gets the `path_match` from the given `trail` map.
```

Comment:
don't say it's a `map`. The type is opaque, so it should say "from the given `trail`"